app/src/main/AndroidManifest.xml

```diff
@@ -62,16 +62,25 @@
             android:configChanges="keyboardHidden|orientation|screenSize"
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme" />
+
         <activity
             android:name=".ui.activity.home.HomeActivity"
             android:configChanges="keyboardHidden|orientation|screenSize"
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme" />
+
         <activity
             android:name=".ui.activity.QRScannerActivity"
             android:configChanges="keyboardHidden|orientation|screenSize"
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme" />
+
+        <activity
+            android:name=".ui.activity.walletcreation.CreateEmojiIdActivity"
+            android:configChanges="keyboardHidden|orientation|screenSize"
+            android:screenOrientation="portrait"
+            android:theme="@style/NoAnimTheme" />
+
```

Comment:
Can we not have the entire onboarding flow as a single `OnboardingFlowActivity` and the various screens are `Fragments` in that activity? @kukabi thoughts?

Reply:
@CjS77 @jigneshsanghani If separate screens are part of a single functionality/feature then I think it's better to gather them under an activity. I think it can be applied to the onboarding flow, yes.